csfieldguide/chapters/content/en/human-computer-interaction/sections/usability-heuristics.md

```diff
@@ -285,7 +285,7 @@ Some controls try to offer the best of both worlds by having a small flap that c
 
 # Scary interfaces
 
-The following site identified some of the "scariest" interfaces around, some of which are great examples of *not* having minimalist design:  [OK/Cancel scariest interface](http://okcancel.com/archives/article/2005/11/the-scariest-interface-part-ii.html).
+The following site identified some of the "scariest" interfaces around, some of which are great examples of *not* having minimalist design:  [OK/Cancel scariest interface](https://web.archive.org/web/20191216061736/http://okcancel.com:80/archives/article/2005/11/the-scariest-interface-part-ii.html).
```

Comment:
It's not really `the following site` anymore, it's an archive site showing a copy of the site which no longer exists, And some of what that archive links to also doesn't exist. Not sure how we should deal with it

Reply:
Will have a chat to Tim